### ISSUE TITLE
Spin Docs website on Cloud

### ISSUE THREAD

**Issue by vdice** [fermyon/spin#1819]
Update deployment of the Spin docs website (at `/docs`, a redirect app from spin.fermyon.dev to developer.fermyon.com/spin) to run on Fermyon Cloud.

**Comment by itowlson** [comment]
@vdice is this fixed via #1820?

**Comment by vdice** [comment]
@itowlson No, this issue is around overhauling the deployment of this website to run on Fermyon Cloud (today, it runs on a separate Nomad cluster directly).

**Comment by itowlson** [comment]
@vdice The issue history says "vdice moved this from TODO to Done in  Spin Triage on Mar 14, 2024."  Does this mean it can be closed, or was that one of our occasional project board weirdnesses?

**Comment by vdice** [comment]
@itowlson thank you for checking on this olde issue.  This can now be closed as the Spin documentation website moved to Fermyon Cloud as part of the CNCF migration earlier this year.
